.github/workflows/ci.yml

```diff
@@ -0,0 +1,213 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "v*"
+  pull_request:
+jobs:
+  linux:
+    strategy:
+      fail-fast: false
+      matrix:
+        compiler: [gcc, clang]
+        image: [ubuntu-20.04, ubuntu-22.04]
+    runs-on: ${{ matrix.image }}
+    env:
+      CC: ${{ matrix.compiler }}
+      SUFFIX: linux-${{ matrix.image}}-${{ matrix.compiler }}
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+      - name: Install packages
+        run: |
+          sudo apt-get update -qq
+          sudo apt-get install -y \
+            automake \
+            autoconf \
+            bison \
+            flex \
+            gdb \
+            python3
+      - name: Build
+        run: |
+          autoreconf -fi
+          ./configure --disable-dependency-tracking \
+            --disable-silent-rules \
+            --disable-maintainer-mode \
+            --disable-valgrind \
+            --with-oniguruma=builtin \
+            --enable-static \
+            --enable-all-static \
+            YACC="$(which bison) -y"
+          make
+      - name: Test
+        run: |
+          make check
+      - name: Upload Test Logs
+        if: ${{ failure() }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: test-logs-${{ env.SUFFIX }}
+          retention-days: 7
+          path: |
+            test-suite.log
+            tests/*.log
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: jq-${{ env.SUFFIX }}
+          if-no-files-found: error
+          retention-days: 7
+          path: |
+            jq
+  macos:
+    strategy:
+      fail-fast: false
+      matrix:
+        compiler: [gcc, clang]
+        image: [macos-11, macos-12, macos-13]
+    runs-on: ${{ matrix.image }}
+    env:
+      CC: ${{ matrix.compiler }}
+      SUFFIX: macos-${{ matrix.image}}-${{ matrix.compiler }}
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+      - name: Install packages
+        run: |
+          # brew update sometimes fails with "Fetching /usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask failed!"
+          brew update || brew update-reset
+          brew install \
+            autoconf \
+            automake \
+            libtool \
+            flex \
+            bison
+          sed -i.bak '/^AM_INIT_AUTOMAKE(\[-Wno-portability 1\.14\])$/s/14/11/' modules/oniguruma/configure.ac
```

Comment:
What was this thing about?

Reply:
This line was inherited from the old CI setup. I believe it was to enforce onigurma's automake to be on the same version as jq. I confirmed this is not needed now. Removing.